utils/update_build_version.py

```diff
@@ -17,16 +17,13 @@
 # Updates an output file with version info unless the new content is the same
 # as the existing content.
 #
-# Args: <changes-file> <output-file>
+# Args: <repo-path> <output-file>
 #
 # The output file will contain a line of text consisting of two C source syntax
 # string literals separated by a comma:
-#  - The software version deduced from the given CHANGES file.
+#  - The software version deduced from the last release tag.
```

Comment:
The `<repo-path>` parameter is no longer mentioned. It might seem useless.  You should reword this to say "from the last release tag in the repository at <repo-path>", or something like that.

Reply:
the repo-path is used to determine where to run the git command from, changed phrasing so it's clear.